README.md

```diff
@@ -357,6 +357,16 @@ The schema of the type will look like the following instead of describing all th
    "format": "uuid"
 }
 ```
+
+If you want to override the `nullable` property of a type, you can implement the `Nullable` interface for this type.
```

Comment:
A link to the code definition of the interface wrapping the `Nullable` inline-code would be nice.

Reply:
Changed it a bit and added a link to https://pkg.go.dev/database/sql#NullString which hopefully makes it a bit more clear how/why this is useful 👍